.github/workflows/ci.yml

```diff
@@ -211,6 +210,5 @@ jobs:
           git config user.name "${{ github.actor }}"
           git config user.email "${{ github.actor }}@users.noreply.github.com"
           git checkout .
-          npm run bump:alpha --preid="${{ github.ref_name }}"
-          npm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
-          npm run publish:alpha
+          yarn bump:alpha --preid="${{ github.ref_name }}"
```

Comment:
how will yarn publish with pout an access token?

Reply:
Didn't always work though... that's why the config was added that way :), so if you see some weird 40x errors when publishing it'll be that :)